src/components/calculator.css

```diff
@@ -0,0 +1,42 @@
+.main {
+  border: 2px solid black;
+  width: 200%;
+  height: 150%;
+  display: flex;
+  flex-direction: column;
+}
+
+.answer {
+  background-color: gray;
+  text-align: end;
+  padding-right: 0.5rem;
+}
+
+.firstbox,
+.secondbox,
+.thirdbox,
+.fourthbox,
+.fifthbox {
+  display: flex;
+  flex-direction: row;
+}
+
+.button1-1,
+.button1-2,
+.button1-3,
+.button1-4,
+.button5-2,
+.button5-4 {
+  width: 25%;
+  padding: 0.5rem;
+}
+
+.button5-1 {
```

Comment:
- [ ] Please notice that as mentioned in the requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/228338507-8f65e917-437f-4757-9dcb-bfc7e0caf038.png)
 We are not suppose to add any extra design, please make sure the calculator matches the given design as much as possible 
![image](https://user-images.githubusercontent.com/97521557/228338680-a61ac0ce-343e-4ca0-a986-6e2d97899e41.png)